modules/client-handler/src/test/java/org/apache/ignite/client/handler/ClientPrimaryReplicaTrackerTest.java

```diff
@@ -52,7 +53,7 @@ class ClientPrimaryReplicaTrackerTest extends BaseIgniteAbstractTest {
     @BeforeEach
     public void setUp() throws Exception {
         driver = new FakePlacementDriver(PARTITIONS);
-        driver.setReplicas(List.of("s1", "s2"), TABLE_ID, 1);
+        driver.setReplicas(List.of("s1", "s2"), TABLE_ID, ZONE_ID, 1);
```

Comment:
Do I understand correctly, that you've also run these tests in the colocation branch?

Reply:
Yes